packages/location-management/src/components/LocationUnitAddEdit/Form.tsx

```diff
@@ -108,7 +110,7 @@ export const onSubmit = async (
 
   let geographicLevel: number | undefined | void;
   if (values.parentId) {
-    geographicLevel = await new OpenSRPService(accessToken, API_BASE_URL, LOCATION_HIERARCHY)
+    geographicLevel = await new OpenSRPService(accessToken, opensrpBaseURL, LOCATION_HIERARCHY)
```

Comment:
is `opensroBaseUrl` the correct naming pattern we should use? @kelvin-muchiri @p-netm @moshthepitt

Reply:
I think it show be `OPENSRP_BASE_URL`

Reply:
@dubdabasoduba Props are usually denoted by camel case, and Constants with UpperCase pattern.

Reply:
oooh okay. Makes sense.